src/NuGet.Indexing/NuGetIndexSearcher.cs

```diff
@@ -62,6 +62,12 @@ public class NuGetIndexSearcher : IndexSearcher
         public OpenBitSet LatestStableBitSet { get; }
         public OwnersResult Owners { get; }
         public DateTime LastReopen { get; }
+        public DateTime LastIndexReloadTime => Manager.LastTimeIndexReloaded;
+        public long LastIndexReloadDurationInMilliseconds => Manager.LastIndexReloadDurationInMilliseconds;
+        public DateTime LastAuxiliaryDataLoadTime => Manager.LastTimeAuxiliaryDataRefreshed;
+        public IDictionary<string, DateTime?> LastAuxiliaryDataUpdateTime => Manager.AuxiliaryDataLastUpdateTime;
```

Comment:
Why is this a dictionary? Don't we know all of the keys at compile time? I think a strongly typed object is more maintainable/discoverable.